var/spack/repos/builtin/packages/verrou/package.py

```diff
@@ -0,0 +1,117 @@
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+import glob
+import os
+import sys
+
+
+class Verrou(AutotoolsPackage):
+    """A floating-point error checker.
+
+    Verrou helps you look for floating-point round-off errors in programs. It
+    implements a stochastic floating-point arithmetic based on random rounding:
+    all floating-point operations are perturbed by randomly switching rounding
+    modes. This can be seen as an asynchronous variant of the CESTAC method, or
+    a subset of Monte Carlo Arithmetic, performing only output randomization
+    through random rounding.
+    """
+
+    homepage = "https://github.com/edf-hpc/verrou"
+    url      = "https://github.com/edf-hpc/verrou.git"
+
+    version('master',
+            git='https://github.com/edf-hpc/verrou.git',
+            branch='master')
+
+    version('2.0.0',
+            git='https://github.com/edf-hpc/verrou.git',
+            commit='a614ad695d060abdfcd3cec11022d6f044a0980d')
+    version('1.1.0',
+            git='https://github.com/edf-hpc/verrou.git',
+            commit='d69787e89507519e14f5ec44150c1882930f692e')
+    version('1.0.0',
+            git='https://github.com/edf-hpc/verrou.git',
+            commit='519dd9a83738f72da8446322e9c403e608940b7a')
+    version('0.9.0',
+            git='https://github.com/edf-hpc/verrou.git',
+            commit='ebabfcf7f4e9a02b130e9819ba6c19e0450eb73d')
+
+    resource(name='valgrind-3.13.0',
+             git='git://sourceware.org/git/valgrind.git',
+             commit='1378ec95f22235e0a8c972cf1cd0abef0b9610d4',
+             when='@1.1.0:,master')
+    resource(name='valgrind-3.12.0',
+             git='git://sourceware.org/git/valgrind.git',
+             commit='36923ab298c8d2791d6c86b6cd5c0667c07449f6',
+             when='@1.0.0:1.0.999')
+    resource(name='valgrind-3.10.1',
+             git='git://sourceware.org/git/valgrind.git',
+             commit='cfc3175a6eb132a5600fcf705f3fa326d71483bf',
+             when='@0.9.0:0.9.999')
+
+    conflicts('valgrind')
+
+    variant('fma', default=True,
+            description='Activates fused multiply-add support for Verrou')
+
+    depends_on('autoconf', type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool', type='build')
+    depends_on('m4', type='build')
```

Comment:
Are these no longer needed then, or is it using Spack's default `autoreconf` method?

Reply:
Spack's default `autoreconf` uses them in this case. (I suspect it calls `autogen.sh` under the hood, as I did manually before)

Reply:
If you're curious, it's defined [here](https://github.com/spack/spack/blob/develop/lib/spack/spack/build_systems/autotools.py#L195).